src/main/java/org/apache/sling/junit/impl/servlet/HtmlRenderer.java

```diff
@@ -23,19 +23,22 @@
 
 import javax.servlet.http.HttpServletResponse;
 
-import org.apache.felix.scr.annotations.Component;
-import org.apache.felix.scr.annotations.Service;
 import org.apache.sling.junit.Renderer;
 import org.apache.sling.junit.RendererFactory;
 import org.apache.sling.junit.TestSelector;
 import org.junit.runner.Description;
 import org.junit.runner.Result;
 import org.junit.runner.notification.Failure;
 import org.junit.runner.notification.RunListener;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.ServiceScope;
 
 /** HTML renderer for JUnit servlet */
-@Component(immediate=false)
-@Service(serviceFactory=true)
+@Component(
+    service = Renderer.class,
+    immediate = false,
+    scope = ServiceScope.BUNDLE
```

Comment:
scope?

Reply:
Fixed in the latest changes. Will do another pull request.